ebean-core/src/main/java/io/ebeaninternal/api/BindParams.java

```diff
@@ -422,6 +421,10 @@ public int hashCode() {
       return hc;
     }
 
+    void queryBindHash(BindHash hash) {
+      hash.update(isInParam).update(isOutParam).update(type).update(inValue);
```

Comment:
CHECKME: I've added queryBindHash here. I think `equals` (some lines below) is not correctly implemented here, as it relies only on hashCode